multimedia/ffmpeg/Portfile

```diff
@@ -9,8 +9,8 @@ PortGroup           active_variants 1.1
 name                ffmpeg
 conflicts           ffmpeg-devel
 epoch               1
-version             4.0.2
-revision            1
+version             4.0.3
+revision            0
```

Comment:
Shouldn't we remove the `revision` line entirely here?

Reply:
Done. Both are acceptable though.

Reply:
I was always told leaving `revision 0` out was better style. I'm only repeating that.